pkg/cli/image/image.go

```diff
@@ -36,6 +37,7 @@ func NewCmdImage(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra
 			Commands: []*cobra.Command{
 				info.NewInfo(f, streams),
 				mirror.NewCmdMirrorImage(streams),
+				convert.NewConvert(f, streams),
```

Comment:
I don't think that `oc image convert` is a good place, I'd suggest moving this as `oc adm migrate icsp`, this will much better suit the work this command is responsible for.

Reply:
When moving, check if you could just re-use the this migrator https://github.com/openshift/oc/blob/master/pkg/cli/admin/migrate/migrator.go, similarly how other commands under `oc adm migrate` use it, but that's not a hard requirement to land this.